src/components/tests/dropdown.component.test.js

```diff
@@ -64,4 +65,96 @@ describe('[DropdownComponent]', () => {
             expect(dropdown.list.attr('style')).toEqual('display: none;');
         });
     });
+
+    describe('[setSelection]', () => {
+        it('Should set one of dropdown items selected via code', async () => {
+            const selectOne = $('#selected');
+            const dnd = $('#dnd');
+            const vamp = $('#vampire');
+
+            dropdown.setSelection('Dugeons&Dragons');
+            expect(dropdown.btn.text()).toEqual(dropdown.getSelectedItem());
+            expect(dnd.hasClass('selected')).toBeTruthy();
+
+            dropdown.setSelection('Vampire 5e');
+            expect(dropdown.btn.text()).toEqual(dropdown.getSelectedItem());
+            expect(vamp.hasClass('selected')).toBeTruthy();
+
+            dropdown.setSelection('Select one');
+            expect(dropdown.btn.text()).toEqual(dropdown.defaultText);
+            expect(selectOne.hasClass('selected')).toBeTruthy();
+        });
+    });
+
+    describe('[getSelectedItem]', () => {
+        it('Should return HTML text value from current selected item', async () => {
+            const dnd = $('#dnd');
+            const vamp = $('#vampire');
+
+            expect(dropdown.getSelectedItem()).toEqual(dropdown.unselectedDefaultText);
+
+            dnd.trigger('click');
+            expect(dropdown.getSelectedItem()).toEqual(dnd.text());
+
+            vamp.trigger('click');
+            expect(dropdown.getSelectedItem()).toEqual(vamp.text());
+        });
+    });
+
+    describe('[enabledState]', () => {
+        it('Should return enabled state default or disalbe if setted', async () => {
+            expect(dropdown.isDisabled()).toBeFalsy();
+
+            dropdown.disable();
+            expect(dropdown.isDisabled()).toBeTruthy();
+
+            dropdown.enable();
+            expect(dropdown.isDisabled()).toBeFalsy();
+        });
+    });
+
+    describe('[requiredState]', () => {
+        const otherDropdown = $.parseHTML(`      
+        <div class="row-dropdown-input container" id="testUnrequired">

```

Comment:
You should use beforeEach or beforeAll for this

Reply:
Done.